API/DTOs/TournamentRequestQueryDTO.cs

```diff
@@ -30,6 +31,49 @@ public class TournamentRequestQueryDTO : IPaginated
     [EnumDataType(typeof(Ruleset))]
     public Ruleset? Ruleset { get; init; }
 
+    /// <summary>
+    /// Filters results for only tournaments with a partially matching name or abbreviation (case insensitive)
+    /// </summary>
+    public string? Name { get; init; }
+
+    /// <summary>
+    /// Filters results for only tournaments that occurred on or after a specified date
+    /// </summary>
+    public DateTime? DateMin { get; init; }
+
+    /// <summary>
+    /// Filters results for only tournaments that occurred on or before a specified date
+    /// </summary>
+    public DateTime? DateMax { get; init; }
+
+    /// <summary>
+    /// Filters results for only matches with a specified verification status
```

Comment:
```suggestion
    /// Filters results for only tournaments with a specified verification status
```
?

Reply:
W COPY PASTE

Reply:
im gonna ? you next time I see you copy paste some shit huh